apps/api/v2/src/modules/organizations/inputs/create-organization-membership.input.ts

```diff
@@ -0,0 +1,22 @@
+import { IsBoolean, IsOptional, IsString, IsInt } from "class-validator";
+
+import { MembershipRole } from "@calcom/prisma/enums";
+
+export class CreateOrgMembershipDto {
+  @IsInt()
+  readonly teamId!: number;
+
+  @IsInt()
+  readonly userId!: number;
+
+  @IsOptional()
+  @IsBoolean()
+  readonly accepted?: boolean = false;
+
+  @IsString()
```

Comment:
Even though the imported "MembershipRole" is defined as const instead of enum, I think we can use @IsEnum(MembershipRole) - i would add this and then try making requests with correct and incorrect membersip role and see if it blocks incorrect roles, cuz right now any string can be passed.